lib/models/pr.js

```diff
@@ -153,70 +153,70 @@ class PR {
     get payload() {
         return this._payload;
     }
-        
+
     set payload(payload) {
         return this._payload = payload;
     }
 
     get installation() {
         return this._installation;
     }
-    
+
     get number() {
         return this._number;
     }
-    
+
     get owner() {
         return this._owner;
     }
-    
+
     get repo() {
         return this._repo;
     }
-    
+
     get id() {
         return this._id;
     }
-    
+
     get body() {
         return this._body = this._body || (this.payload.body || "").replace(/\r\n/g, "\n").trim();
     }
-    
+
     get processor() {
         return this.config.type.toLowerCase();
     }
-    
+
     get aws_s3_bucket() {
         if (process.env.ALLOW_MULTIPLE_AWS_BUCKETS == "no") {
             return process.env.AWS_BUCKET_NAME;
         }
         return this.owner == "whatwg" ? process.env.WHATWG_AWS_BUCKET_NAME : process.env.AWS_BUCKET_NAME;
     }
-    
+
     get isMultipage() {
         return !!this.config.multipage;
     }
 
     requiresPreview() {
         return !(/<!--\s*no preview\s*-->/.test(this.body));
     }
-    
+
     touchesSrcFile() {
         let relFiles = this.relevantSrcFiles;
         return this.files.some(f => relFiles.indexOf(f.filename) > -1);
     }
-    
+
     get relevantSrcFiles() {
-        return [this.config.src_file];
+        return [this.config.src_file] + this.config.includes;
```

Comment:
```suggestion
        return [this.config.src_file].concat(this.config.includes);
```
Python much? :smile:

Reply:
Heh, 😊, a test caught that for me somewhere else, but this function must not be tested. I re-learned the other day that PHP uses `.` for string concatenation, but I'm sure I'll forget it again by the next time I need it.

Reply:
There are very few tests because there is very little budget. :)